app/api.js

```diff
@@ -137,7 +137,12 @@ async function upload(
   const host = window.location.hostname;
   const port = window.location.port;
   const protocol = window.location.protocol === 'https:' ? 'wss:' : 'ws:';
-  const ws = await asyncInitWebSocket(`${protocol}//${host}:${port}/api/ws`);
+  const endpoint =
+    window.location.protocol === 'file:'
+      ? 'wss://send2.dev.lcip.org/api/ws'
```

Comment:
How should we do this for real? A build-time variable in webpack maybe?

Reply:
Good question. I created an issue to track this. https://github.com/mozilla/send/issues/869

We should do it however other similar configuration of this type is handled. A build-time variable in webpack seems like the right thing. Are there other things configured in this way already?